File: specification/specs2/indices/status_management/clear_cache/ClearCacheRequest.ts
class ClearCacheRequest extends RequestBase {
	@request_parameter()
	fielddata: boolean;
	@request_parameter()
	fields: Field[];
	@request_parameter()
	query: boolean;
	@request_parameter()
	ignore_unavailable: boolean;
	@request_parameter()
	allow_no_indices: boolean;
	@request_parameter()
	expand_wildcards: ExpandWildcards;
	@request_parameter()
	recycler: boolean;
	@request_parameter()
	request_cache: boolean;
	@request_parameter()
	request: boolean;
}

File: output/csharp/Indices/StatusManagement/ClearCache/ClearCacheRequest.cs
using Nest.Common;
using Nest.CommonAbstractions;
using Nest.Internal;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.Indices {

	public class ClearCacheRequest : RequestBase {
		
		[DataMember(Name="allow_no_indices")]
		public bool AllowNoIndices { get; set; }


		[DataMember(Name="expand_wildcards")]
		public ExpandWildcards ExpandWildcards { get; set; }


		[DataMember(Name="fielddata")]
		public bool Fielddata { get; set; }


		[DataMember(Name="fields")]
		public List<Field> Fields { get; set; }


		[DataMember(Name="ignore_unavailable")]
		public bool IgnoreUnavailable { get; set; }


		[DataMember(Name="query")]
		public bool Query { get; set; }


		[DataMember(Name="request")]
		public bool Request { get; set; }

	}
}
